src/internal/SDKAnalyticsConstants.ts

```diff
@@ -0,0 +1,29 @@
+import { Platform } from 'react-native';
+
+const getReactNativeVersion = () => {
+  try{
+    const version = Platform.Version;
+    return version.toString();
+  } catch {
+    return "0.0.0"
+  }
+}
+
+const getSDKVersion = () => {
+  try{
+    const SDKVersionPackageJson = require('cloudinary-react-native/package.json')
```

Comment:
Don't we want to have a script that populates this SDK version inside `SDKAnalyticsConstants` (like we have in other FE SDK's)

Reply:
I'll check, though I'm not sure why a script is required here

Reply:
Maybe the idea behind it was to not load `package.json` on the customer's side... but i cannot think of other good reason to do that

Reply:
I could not find a justification to create a script here, unlike the FEF we only have 1 package here who make use of extracting the version, if we find there's a need for a script we can write one in the future.